src/js/modules/scrollToAnchor.js

```diff
@@ -0,0 +1,22 @@
+import { closeMenu } from './mobileMenu.js';
+
+const anchors = Array.from(document.querySelectorAll("a[href*='#']"));
+
+anchors.forEach((anchor) => {
+	anchor.addEventListener("click", (event) => {
+		event.preventDefault();
+		closeMobileMenu();
+		const blockID = anchor.getAttribute("href");
+		document.querySelector(`${blockID}`).scrollIntoView({
+			behavior: "smooth",
+			block: "start"
+		})
+	})
+})
+
+function closeMobileMenu() {
+	const menuOverlay = document.body.querySelector(".header__menu-overlay");
```

Comment:
when you move code, use existing variable

Reply:
It doesn't need to be done after moving all this code into menu.js